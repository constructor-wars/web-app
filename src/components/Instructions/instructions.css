.instructions__text {
  letter-spacing: .5px;
  padding: 10px;
}
