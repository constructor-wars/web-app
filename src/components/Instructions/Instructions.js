import React from "react";
import "./instructions.css";
import Markdown from "react-remarkable";

class Instructions extends React.Component {
  render() {
    return (
      <div className="instructions__text">
        <h1>{this.props.question_title}</h1>
        <Markdown source={this.props.instructions} />
      </div>
    );
  }
}

export default Instructions;
