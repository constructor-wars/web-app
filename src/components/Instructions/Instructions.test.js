import React from "react";
import Instructions from "./Instructions";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

describe("Instructions", () => {
  it("matches the snapshot", () => {
    const tree = renderer
      .create(<Instructions data="MockInstructions" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("tests instructions component", () => {
    const wrapper = shallow(<Instructions data="MockInstructions" />);
    const text = wrapper.find("div").text();
    expect(text).toEqual("MockInstructions");
  });
});
