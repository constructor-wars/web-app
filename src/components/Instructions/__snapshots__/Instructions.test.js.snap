// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instructions matches the snapshot 1`] = `
<div
  className="instructions__text"
>
  <div />
</div>
`;
